project/frontend/src/components/auth/LogoutButton.tsx

```
@@ -0,0 +1,20 @@
+
+export const LogoutButton = () => {
+  const handleLogout = useCallback(() => {
+    Cookies.remove("token");
+    keycloak.logout();
+  }, []);
+
+  return (
+    <button
+      onClick={handleLogout}
+      className="px-4 py-2 bg-red-500 text-white rounded hover:bg-red-600"
+    >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)